eventq_aws/lib/eventq_aws/aws_queue_manager.rb

```diff
@@ -8,7 +8,7 @@ class QueueManager
       def initialize(options)
 
         if options[:client] == nil
```

Comment:
Sorry can we make this check the same as the `valid_client?(options)` check in the queue_worker?